v2/cli/term/log_page.go

```diff
@@ -0,0 +1,161 @@
+package term
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/armon/circbuf"
+	"github.com/brigadecore/brigade/sdk/v2/core"
+	"github.com/gdamore/tcell/v2"
+	"github.com/rivo/tview"
+)
+
+const logPageName = "log"
+
+type logPage struct {
+	*page
+	logText  *tview.TextView
+	logBuf   *circbuf.Buffer
+	maxBytes int64
+}
+
+func newLogPage(
+	apiClient core.APIClient,
+	app *tview.Application,
+	router *pageRouter,
+) *logPage {
+	l := &logPage{
+		page:    newPage(apiClient, app, router),
+		logText: tview.NewTextView().SetDynamicColors(true),
+	}
+
+	l.maxBytes = 65535
```

Comment:
For this and the logic around this max (truncating text, etc.), did we just want to align with how we present log text in the GitHub gateway (GH limits check text bodies to this byte maximum)?  Otherwise, we should be fine to present a larger amount of logs sans truncation -- or introduce pagination, etc.

Reply:
That is one reason. However, the main reason for implementing the circular buffer is that when there are a large amount of logs, we don't want to store a massive string in memory, which may cause memory issues.

Reply:
Ah yes, thank you for the reminder.  So perhaps a follow-up could be to utilize the same circular buffer but instead of truncation, add pagination to the logs window (if needed).

Reply:
The API only streams logs-- it doesn't have any paging capability. If we needed to add that, that's a relatively complex addition to the API. This isn't to say it cannot be done, but only that we should consider it only after the need is very clear.